engine/table/src/main/java/io/deephaven/engine/table/impl/remote/ConstructSnapshot.java

```diff
@@ -1191,6 +1191,19 @@ public static long callDataSnapshotFunction(
             @NotNull final SnapshotFunction function) {
         final long overallStart = System.currentTimeMillis();
         final StateImpl state = StateImpl.get();
+        final UpdateGraph updateGraph = control.getUpdateGraph();
+
+        if (updateGraph == null) {
+            // This is a snapshot of static data. Just call the function with no frippery.
+            final boolean functionSuccessful = function.call(false, LogicalClock.NULL_CLOCK_VALUE);
+            Assert.assertion(functionSuccessful, "functionSuccessful");
+            if (log.isDebugEnabled()) {
+                final long duration = System.currentTimeMillis() - overallStart;
+                log.debug().append(logPrefix)
+                        .append(" Static snapshot function elapsed time ").append(duration).append(" ms").endl();
+            }
+            return LogicalClock.NULL_CLOCK_VALUE;
+        }
```

Comment:
I think b/c you are looking at changes since your last review, and that includes a merge + resolve from main; if you look at the full PR, you won't see this.